packages/angular_devkit/schematics/src/utility/linked-list.ts

```diff
@@ -6,6 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+/** @deprecated Since v15.0 */
 export class LinkedList<T extends { next: T | null }> {
```

Comment:
Can be removed since this is private API.

Reply:
Removed it 👍